Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -1242,15 +1206,14 @@
     ]
   },
   {
-    "Input": "Help me alsjeblieft een meeting te regelen van 1:30 tot 15:00",
+    "Input": "Help me alsjeblieft een meeting te regelen van 1:30 tot 3 's middags",
```

Comment:
Shouldn't we keep both? Assuming the previous is not a duplicate?
BTW, in future PRs, if edit are due to duplication, can you please list the cases in the PR description or as comments left on GitHub before review?
Thanks!

Reply:
PR also has conflicts with /master, @LionbridgeCSII

Reply:
added spec